src/DotNetLightning.Core/Serialization/Msgs/Msgs.fs

```diff
@@ -272,6 +276,11 @@ module ILightningSerializable =
 
             (d :> ILightningSerializable<ErrorMsg>)
                 .Serialize(ls)
+        | :? WarningMsg as d ->
```

Comment:
@webwarrior-ws please use a longer var name than 'd'

Reply:
that name is used in all other cases